Application/EdFi.Ods.Api/Startup/OdsStartupBase.cs

```diff
@@ -118,6 +121,17 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddAuthentication(EdFiAuthenticationTypes.OAuth)
                 .AddScheme<AuthenticationSchemeOptions, EdFiOAuthAuthenticationHandler>(EdFiAuthenticationTypes.OAuth, null);
+     
+            if (ApiSettings.IsFeatureEnabled(ApiFeature.IdentityManagement.GetConfigKeyName()))
```

Comment:
You've added a new Feature, but none of the work to wrap the rest of the feature has been done.  We need to either fully extract Identity Management into a feature, or spin off a separate ticket to manage that and instead just register the authorization all the time here.